daemons/validator_score_pythnet_v1_daemon.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require_relative "../config/environment"
+include ValidatorScoreV1Logic
+include PipelineLogic
+
+# Send an interrupt with `ctrl-c` or `kill` to stop the script. Results will
+# not be posted to the validators.app server.
+interrupted = false
+trap("INT") { interrupted = true }  unless Rails.env.test?
+
+network = "pythnet"
+sleep_time = 15 # seconds
```

Comment:
According to https://www.pivotaltracker.com/story/show/183730152
```
Pythnet network should behave in the same way as testnet.
```

Should not we change this value to `60` ?